src/shared/Core/Trace2Message.cs

```diff
@@ -175,19 +179,21 @@ private static string BuildSpan(PerformanceFormatSpan component, string data)
 
 public class VersionMessage : Trace2Message
 {
-    [JsonProperty("evt")]
+    [JsonPropertyName("evt")]
+    [JsonPropertyOrder(8)]
     public string Evt { get; set; }
 
-    [JsonProperty("exe")]
+    [JsonPropertyName("exe")]
+    [JsonPropertyOrder(9)]
     public string Exe { get; set; }
 
     public override string ToJson()
     {
-        return JsonConvert.SerializeObject(this,
-                new StringEnumConverter(typeof(SnakeCaseNamingStrategy)),
-            new IsoDateTimeConverter()
+        return JsonSerializer.Serialize(this,
+            new JsonSerializerOptions
             {
-                DateTimeFormat = TimeFormat
+                PropertyNameCaseInsensitive = true,
+                Converters = { new JsonStringEnumConverter(new SnakeCaseNamingPolicy()) }
```

Comment:
Could we make the `JsonSerializerOptions` a `protected static readonly` instance on the abstract `Trace2Message`, to avoid needing to construct a new one for each message type?